src/index.html

```diff
@@ -1,12 +1,425 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a href="#home">
+            <img
+              class="header__img"
+              src="./images/logo.png"
+              alt="logo"
+              >
```

Comment:
```suggestion
            <img
              class="header__img"
              src="./images/logo.png"
              alt="logo"
            >
```
pay attention to indentation (checklist)